src/command.c

```diff
@@ -128,12 +128,12 @@ bool debug_bmp;
 #endif
 unsigned cortexm_wait_timeout = 2000; /* Timeout to wait for Cortex to react on halt command. */
 
-int command_process(target_s *t, char *cmd)
+int command_process(target_s *const t, char *const cmd_data)
```

Comment:
The `for` loop inside looking for a matching command handler shadowed this `cmd`, I chose  `cmd_data` because this contained the command and the arguments, which are tokenised inside, that said, it's not exactly a better choice